2_set_secrets.sh

```diff
@@ -2,21 +2,14 @@
 
 api_key=$(docker-compose exec conjur rails r "print Credentials['cucumber:user:admin'].api_key")
 
-echo '--------- Set Secrets defined in Policy ------------'
-docker-compose run --rm -e CONJUR_AUTHN_API_KEY=$api_key conjur_cli \
-  variable values add staging/myapp/database/username foo_staging_user
-
-docker-compose run --rm -e CONJUR_AUTHN_API_KEY=$api_key conjur_cli \
-  variable values add staging/myapp/database/password "$(openssl rand -hex 12)"
-
-docker-compose run --rm -e CONJUR_AUTHN_API_KEY=$api_key conjur_cli \
-  variable values add staging/myapp/stripe/private_key "$(openssl rand -hex 60)"
-
-docker-compose run --rm -e CONJUR_AUTHN_API_KEY=$api_key conjur_cli \
-  variable values add production/myapp/database/username foo_production_user
-
-docker-compose run --rm -e CONJUR_AUTHN_API_KEY=$api_key conjur_cli \
-  variable values add production/myapp/database/password "$(openssl rand -hex 12)"
-
-docker-compose run --rm -e CONJUR_AUTHN_API_KEY=$api_key conjur_cli \
-  variable values add production/myapp/stripe/private_key "$(openssl rand -hex 60)"
+command=$(cat <<EOF
+conjur variable values add staging/myapp/database/username foo_staging_user
+conjur variable values add staging/myapp/database/password "staging_$(openssl rand -hex 12)"
+conjur variable values add staging/myapp/stripe/private_key "staging_$(openssl rand -hex 60)"
+
+conjur variable values add production/myapp/database/username foo_production_user
+conjur variable values add production/myapp/database/password "production_$(openssl rand -hex 12)"
+conjur variable values add production/myapp/stripe/private_key "production_$(openssl rand -hex 60)"
+EOF
+)
+docker-compose run --rm -e CONJUR_AUTHN_API_KEY=$api_key --entrypoint="bash -c '$command'" conjur_cli
```

Comment:
Another option to format this is: 
```
docker-compose run --rm -e CONJUR_AUTHN_API_KEY=$api_key --entrypoint /bin/bash conjur_cli -c "
  conjur variable values add staging/myapp/database/username foo_staging_user
  conjur variable values add staging/myapp/database/password "staging_$(openssl rand -hex 12)"
  conjur variable values add staging/myapp/stripe/private_key "staging_$(openssl rand -hex 60)"
  conjur variable values add production/myapp/database/username foo_production_user
  conjur variable values add production/myapp/database/password "production_$(openssl rand -hex 12)"
  conjur variable values add production/myapp/stripe/private_key "production_$(openssl rand -hex 60)
"

Reply:
thank you! i've been using `cat` here, `cat` there, `cat` everywhere + all this `EOF` stuff. i'd somehow convinced myself that^ wouldn't work.